test/e2e/lib/pages/frontend/comments-area-component.js

```diff
@@ -16,50 +15,46 @@ export default class CommentsAreaComponent extends AsyncBaseContainer {
 		super( driver, By.css( '#comments.comments-area' ) );
 	}
 
-	async _postComment( { comment } ) {
-		const commentForm = By.css( '#commentform' );
+	/**
+	 * Posts the first comment to a post (under "Join the Conversation" in the footer).
+	 *
+	 * @param { string } comment The comment text.
+	 */
+	async _postComment( comment ) {
```

Comment:
Wondering if it'd make sense to rename this function to `postComment`. Does anyone know the reason for the `_` prefix?

Reply:
Probably to view it as a private method? 🤷‍♂️ Anyway, I'm for removing the underscore prefix 👍